### ISSUE TITLE
Spring Boot Dashboard - Setting for Cert store with VSCode extension

### ISSUE THREAD

**Issue by Noahhaines171** [microsoft/vscode-spring-boot-dashboard#170]
# Summary

Im hitting an internal application endpoint, through our company proxy. The website has a self signed cert. When I run my spring boot application using the Spring Boot Dashboard, the request to hit this internal website fails with the below error. When I run the application by simply using the `mvn spring-boot:run` command, the request works successfully. 

Is there a specific setting/configuration for pointing the Spring Boot Dashboard to the JDK trust store?

# Error
2022-05-20 12:59:19.928 | [http-nio-8080-exec-1] | ERROR | c.n.m.service.EdgeLayerService | tx.id=1961e414-f514-4181-af8d-82e2e220514a | log: I/O error on POST request for "<InternalApplicationEndpoint>": PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target; nested exception is javax.net.ssl.SSLHandshakeException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target

# Troubleshooting Attempts
I have attempted to set different values for the extension to recognize the Cert Store, to no luck. I've messed with the `.vscode/launch.json` file and attempted to set command line arguments like `-Djavax.net.ssl.certStore=$JAVA_HOME/lib/security/cacerts` and others, but cannot get it resolved.

I've also tried setting these settings in my `settings.json` file:
```
  "spring-boot.ls.java.vmargs": [
    "-Dtrust_all_cert=true",
    "-Dcom.sun.management.jmxremote.ssl=false",
    "-Dcom.sun.net.ssl.checkRevocation=false",
    "-Djavax.net.ssl.certStore=$JAVA_HOME/lib/security/cacerts"
  ],
  "http.proxyStrictSSL": false
  ```

**Comment by Eskibear** [comment]
> Is there a specific setting/configuration for pointing the Spring Boot Dashboard to the JDK trust store?

No. Spring Boot Dashboard simply passes launch.json to Java Debugger extension to launch your app, and by default VS Code's proxy settings should be applied.

I'm not sure what's the trick inside 'spring-boot:run`, but setting `spring-boot.ls.java.vmargs` is about vmargs used to launch Spring Tools language server, which is irrelevant to your app. You might try adding vmargs in launch.json.
